### DISCUSSION TITLE
Name of our project.

### DISCUSSION THREAD

**Discussion by BerkeArgin** [bounswe/2021SpringGroup1#111] [Ideas]
We need to come up with a name for our project. We can use this discussion thread for brainstorming.

**Comment by BerkeArgin** [comment]
My idea for the name is "ProtoPost", Proto prefix is to resemble "prototype" or "protocol". Since our apps main feature is to have user-created, specific and customizable post formats in a community and this is somewhat similar to creating prototypes and designing protocols. I think it may be catchy but seems a little bit technical and business related application rather than a casual social media platform.

**Comment by orkana** [comment]
**neaty**:  a pun of community and a neat platform to see your interest

**nitty**: same pun but instead it's a little more goofy  

**Kamon**: Japanese for "family/clan", also inviting and exciting because it's phonetically "come on"

**Reply by BerkeArgin** [reply]
What about "Definity" , pun on define and community?

**Comment by orkana** [comment]
![First draft](https://user-images.githubusercontent.com/36167517/140913003-aa82172f-c3c5-49eb-bd4d-315c08b11319.png)

**Reply by orkana** [reply]
first draft

**Reply by orkana** [reply]
Second draft; removed the bottom arc, changed the color and added outline
![image](https://user-images.githubusercontent.com/36167517/140914351-353ee344-3ded-4f7f-a101-b7348998bed9.png)

**Comment by yunus-topal** [comment]
I think the design we used in our Demo is pretty good. We can use it if you also agree.
